awx/main/migrations/0012_move_deprecated_stdout.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-12-11 19:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0011_blank_start_args'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DeprecatedJobStdout',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('result_stdout_text', models.TextField(blank=True, default=b'', editable=False)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='deprecatedjobstdout',
+            name='job',
+            field=models.OneToOneField(editable=False, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='deprecated_result_stdout', to='main.UnifiedJob'),
+        ),
```

Comment:
I don't see why there should be 2 separate operations here. Could you roll the `deprecatedjobstdout` field creation into the `CreateModel` by deleting these and running `makemigrations` again?

Reply:
Yea, I was confused by this, too, but it's what `makemigrations` created :/

Reply:
I checked out your branch, and that method gave me

```python
# -*- coding: utf-8 -*-
# Generated by Django 1.11.7 on 2017-12-12 14:44
from __future__ import unicode_literals

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('main', '0013_auto_20171212_1444'),
    ]

    operations = [
        migrations.CreateModel(
            name='DeprecatedJobStdout',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('result_stdout_text', models.TextField(blank=True, default=b'', editable=False)),
                ('job', models.OneToOneField(editable=False, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='deprecated_result_stdout', to='main.UnifiedJob')),
            ],
        ),
    ]
```

Reply:
That's so bizarre O_O, let me give this another look.